tests/readwrite_test.py

```diff
@@ -108,7 +118,9 @@ def test_read_write_data_file(tmp_path):
         pdbx.dump(data_list, output_path)
 
 
-ROUNDTRIPPABLE_CIF_STR = '''data_foo
+# TODO: 2020/07/16 intendo - are the spaces at the end of the line needed
```

Comment:
This string matches the output of the `PdbxWriter`, so the spaces are needed unless we improve `PdbxWriter` to not output trailing space.

Alternative to a `noqa` which also makes flake8 happy: Replace each trailing space with `\x20`.

Reply:
I would prefer to leave this as is for now and create an issue for removing trailing whitespace from PdbxWriter.

Reply:
Created Issue to remove trailing whitespaces in PdbxWriter.